app/assets/javascripts/exchanges.js

```diff
@@ -1,2 +1,4 @@
 // Place all the behaviors and hooks related to the matching controller here.
 // All this logic will automatically be available in application.js.
+var hema = hema || {};
+hema.Tournaments = {};
```

Comment:
This should be `hema.Exchanges = {}` obviously. But it doesn't do anything either way, so we can ignore this.